docs/PRIVATE.md

```diff
@@ -0,0 +1,245 @@
+## Modules
+
+<dl>
+<dt><a href="#module_deriver-DeriverUtils">deriver-DeriverUtils</a> ℗</dt>
+<dd><p>Scoped key deriver utilities</p>
+</dd>
+<dt><a href="#module_deriver-ScopedKeys">deriver-ScopedKeys</a></dt>
+<dd><p>Used by the Firefox Accounts content server</p>
+</dd>
+<dt><a href="#module_relier-KeyUtils">relier-KeyUtils</a> ℗</dt>
+<dd><p>Scoped key utilities</p>
+</dd>
+<dt><a href="#module_relier-OAuthUtils">relier-OAuthUtils</a></dt>
+<dd><p>OAuth methods to authenticate with Firefox Accounts and get scoped keys</p>
+</dd>
+<dt><a href="#module_relier-util">relier-util</a> ℗</dt>
+<dd><p>Relier utility functions</p>
+</dd>
+</dl>
+
+<a name="module_deriver-DeriverUtils"></a>
+
+## deriver-DeriverUtils ℗
+Scoped key deriver utilities
+
+**Access**: private  
+<a name="module_deriver-DeriverUtils..encryptBundle"></a>
+
+### deriver-DeriverUtils~encryptBundle(appPublicKeyJwk, bundle) ⇒ <code>Promise</code>
+**Kind**: inner method of [<code>deriver-DeriverUtils</code>](#module_deriver-DeriverUtils)  
+
+| Param | Type |
+| --- | --- |
+| appPublicKeyJwk | <code>string</code> | 
+| bundle | <code>object</code> | 
+
+<a name="module_deriver-ScopedKeys"></a>
+
+## deriver-ScopedKeys
+Used by the Firefox Accounts content server
+
+**Example**  
+```js
+const scopedKeys = new fxaCryptoDeriver.ScopedKeys();
+
+return scopedKeys.deriveScopedKey({
+  identifier: 'https://identity.mozilla.com/apps/notes',
+  inputKey: 'bc3851e9e610f631df94d7883d5defd5e5f55ab520bd5a9ae33dae26575c6b1a',
+  keyMaterial: '0000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
My comment from https://github.com/mozilla/fxa-content-server/pull/5675/files#r150135944 applies here as well, as does my :-/ face about the difficulty of naming this thing...

Reply:
for the two above:

`keyRotationSecret`
`keyRotationTimestamp`